CrowdStrike/crowdstrike_falcon/ingest/parser.yml

```diff
@@ -132,6 +132,10 @@ stages:
           threat.indicator.type: "file"
         filter: "{{parsed_event.message.event.IOCType == 'hash_sha256'}}"
 
+      - set:
+          process.parent.executable: "{{parsed_event.message.event.ParentImageFilePath}}"
+        filter: '{{parsed_event.message.metadata.eventType == "EppDetectionSummaryEvent"}}'
```

Comment:
Condition is here because `DetectionSummaryEvent` [seems to be having ](https://github.com/SEKOIA-IO/intake-formats/blob/main/CrowdStrike/crowdstrike_falcon/tests/detection_summary_event1.json#L50) a full path in `ParentImageFileName`